### DISCUSSION TITLE
Close Map button on rightmost tab too close to Library button

### DISCUSSION THREAD

**Discussion by Rycochet** [encounterplus/encounterplus#491] [Ideas]
It's too easy to accidentally close a map when trying to click the Library button - needs some thought about what to do instead, but capturing here.

Potential options:

* Put the X in a different corner.
* Make it a `...` dropdown with multiple options such as Close and Duplicate.
* Ensure there's a "Are you sure?` confirmation (if there isn't already).

...?

**Comment by HighFiv-e** [comment]
if "are you sure" is implemented, please allow a toggle for this
